src/components/app/list/AppListView.tsx

```diff
@@ -256,7 +257,7 @@ export class AppListView extends Component<AppListViewProps> {
                                                     className="default-tt"
                                                     arrow={true}
                                                     placement="top"
-                                                    content={app.defaultEnv.lastDeployedTime}
+                                                    content={moment(app.defaultEnv.lastDeployedTime).format(DATE_TIME_FORMATS.TWELVE_HOURS_FORMAT)}
```

Comment:
If possible try to use handleUTCTime method here

Reply:
Need time in 12 hour format,